packages/driver/src/cy/keyboard.ts

```diff
@@ -801,6 +803,7 @@ export class Keyboard {
     let eventConstructor = 'KeyboardEvent'
     let cancelable = true
     let addModifiers = true
+    let inputType = ''
```

Comment:
I think this will add the `inputType` to every event since it's not initialized as `undefined`.  Since we don't test for extra properties we should add a test for that.

I'll take a look

Reply:
You're right. It should be written in that way.